README.md

```diff
@@ -14,9 +14,12 @@ A web tool to explore the data is available [here](https://telemetry.mozilla.org
 ## Adding a New Glean Repository
 
 To scrape a git repository for probe definitions, an entry needs to be added in `repositories.yaml`.
+Each application repository should be listed under `application_families`.
```

Comment:
This is the first mention of `application_families`. Probably fine for people that already knwo about it, but it might not be clear to everyone that a "family" is "same ~code base, but different app ids/builds/..."

I also don't know how to structure this better at the moment, so consider this not fully-blocking, but maybe some future improving/rewording needed.